src/models/repository.model.ts

```diff
@@ -338,6 +338,13 @@ export default class Repository extends Model implements IRepository {
 
         Repository.openAuthorizeDialog(this.entity)
       }
+      else if (e.response?.status === 400) {
+        CzNotification.toast({
+          message: e.response?.data,
```

Comment:
@pkdash - I think we need to avoid proxying error messages from the back-end and instead provide a more user-friendly message.

Reply:
@Maurier You mean just hard code the error message here something like the "Temporal coverage end date must be equal to or greater than the start date"? In that case we have to assume that the only error in parsing the submitted data at the backend is the validation of temporal coverage.